releasenotes/notes/cws-7-54-bb2551a1d930853b.yaml

```
@@ -0,0 +1,22 @@
+---
+features:
+  - |
+    CWS: Add support for ``write`` events on Windows.
+  - |
+    CWS: Add support for ``delete`` event on Windows.
+  - |
+    CWS: Add ``chdir``, ``mount`` and ``umount`` in ``ptracer`` mode.
```

Verifying if it's supposed to be `unmount` here?

```suggestion
    CWS: Add ``chdir``, ``mount``, and ``unmount`` in ``ptracer`` mode.
```

---

The event is named umount but you're right, it should be unmount.